pkg/descheduler/evictions/evictions.go

```diff
@@ -97,19 +98,21 @@ func (pe *PodEvictor) TotalEvicted() int {
 // EvictPod returns non-nil error only when evicting a pod on a node is not
 // possible (due to maxPodsToEvictPerNode constraint). Success is true when the pod
 // is evicted on the server side.
-func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node, reasons ...string) (bool, error) {
+func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, node *v1.Node, strategy string, reasons ...string) (bool, error) {
 	var reason string
 	if len(reasons) > 0 {
-		reason = " (" + strings.Join(reasons, ", ") + ")"
+		reason = strategy + " (" + strings.Join(reasons, ", ") + ")"
 	}
```

Comment:
sorry, I just noticed the `if len(reasons) > 0` check here... maybe instead we should initialize `reason := strategy` (instead of nil) above this and then append the reasons inside this check